pkg/service/service_binding_builder.go

```diff
@@ -0,0 +1,198 @@
+package service
```

Comment:
Please indicate that this code is a fork from the Service Binding Operator, and what are the conditions to start using the upstream library again (an issue number?)